src/app/features/courses/pipes/minutes-to-hours.pipe.mock.ts

```diff
@@ -0,0 +1,8 @@
+import {Pipe, PipeTransform} from '@angular/core';
+
+@Pipe({name: 'minutesToHours'})
+export class MinutesToHoursMockPipe implements PipeTransform {
+    transform(value: number): string {
+        return '';
+    }
+}
```

Comment:
Where are you use it?

Reply:
The pipe is used in the courses item component template.
the mock is used in the following spec: 
src\app\features\courses\components\courses-item\courses-item.component.spec.ts